plugins/kubernetes-actions/README.md

```diff
@@ -14,63 +14,6 @@ Run the following command to install the action package in your Backstage projec
 yarn workspace backend add @janus-idp/backstage-scaffolder-backend-module-kubernetes
 ```
 
-### Installing the action on the legacy backend
-
-1. [Register](https://backstage.io/docs/features/software-templates/writing-custom-actions#registering-custom-actions) the Kubenretes actions by modifying the `packages/backend/src/plugins/scaffolder.ts` file from your project with the following changes:
-
-   ```ts
-   import { CatalogClient } from '@backstage/catalog-client';
-   import { ScmIntegrations } from '@backstage/integration';
-   import {
-     createBuiltinActions,
-     createRouter,
-   } from '@backstage/plugin-scaffolder-backend';
-
-   import { Router } from 'express';
-
-   import { createKubernetesNamespaceAction } from '@janus-idp/backstage-scaffolder-backend-module-kubernetes';
-
-   import type { PluginEnvironment } from '../types';
-
-   export default async function createPlugin(
-     env: PluginEnvironment,
-   ): Promise<Router> {
-     const catalogClient = new CatalogClient({
-       discoveryApi: env.discovery,
-     });
-
-     const integrations = ScmIntegrations.fromConfig(env.config);
-
-     const builtInActions = createBuiltinActions({
-       integrations,
-       catalogClient,
-       config: env.config,
-       reader: env.reader,
-     });
-
-     const actions = [
-       ...builtInActions,
-       createKubernetesNamespaceAction(catalogClient),
-     ];
-
-     return await createRouter({
-       actions,
-       logger: env.logger,
-       config: env.config,
-       database: env.database,
-       reader: env.reader,
-       catalogClient,
-       identity: env.identity,
-     });
-   }
-   ```
-
-2. **Optional**: If you are doing the previous step for the first time, you also have to install the `@backstage/integration` package
-
-   ```bash
-   yarn workspace backend add @backstage/integration
-   ```
-
 ### Installing the action on the new backend
```

Comment:
```suggestion
### Installing the action on the backend
```

Reply:
I would maybe leave this as it is, if you are ok with it. To stay consistent with other plugin READMEs for now. Or we can remove this from all plugins.

Reply:
I believe we should make the change regardless. Right now, half of our READMEs include "new" and half don't. 😢 We'll definitely need to review all the READMEs in the future.

Reply:
Alright, makes sense :)